core/src/main/java/bio/terra/pearl/core/service/participant/FamilyEnrolleeService.java

```diff
@@ -20,15 +22,23 @@ public FamilyEnrolleeService(FamilyEnrolleeDao familyEnrolleeDao,
     }
 
     @Transactional
-    public void deleteByFamilyId(UUID familyId) {
-        dao.deleteByFamilyId(familyId);
+    public void deleteByFamilyId(UUID familyId, DataAuditInfo info) {
+        List<FamilyEnrollee> objs = dao.findByFamilyId(familyId);
+
+        bulkDelete(objs, info);
     }
 
     @Transactional
-    public void deleteByEnrolleeId(UUID enrolleeId) {
-        dao.deleteByEnrolleeId(enrolleeId);
-    }
+    public void deleteByStudyEnvironmentId(UUID studyEnvironmentId, DataAuditInfo info) {
+        List<FamilyEnrollee> objs = dao.findByStudyEnvironmentId(studyEnvironmentId);
 
+        bulkDelete(objs, info);
+    }
 
+    @Transactional
+    public void deleteByEnrolleeId(UUID id, DataAuditInfo info) {
+        List<FamilyEnrollee> objs = dao.findByEnrolleeId(id);
 
+        bulkDelete(objs, info);
+    }
 }
```

Comment:
Is there a better way to do this in an audited fashion? I don't imagine it's performance sensitive but it was easy to forget to do the auditing (and even get it approved and merged in)

Reply:
confirming what we just talked about, these operations did not technically need to be audited, as they only occur in the context of enrollee deletion or study repopulation